src/frontend/components/app/drawer-portal.tsx

```diff
@@ -74,11 +74,11 @@ const getOrCreatePortalContainer = (id: string) => {
  */
 export const DrawerPortal: React.FC<DrawerPortalProps> = ({ children, width }) => {
   const [drawerElement, setDrawerElement] = useState(document.getElementById(DRAWER_PORTAL_ID))
-  const { to = null } = useSelector<ReduxState, RouterInState>((state) => state.router)
+  const { from = null } = useSelector<ReduxState, RouterInState>((state) => state.router)
```

Comment:
Are you sure `from` will work? If I remember correctly, there was some kind of double navigation and `from` wasn't the latest route here 🤔